lib/wallet/widgets/sign_up_button.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:ax_dapp/wallet/bloc/wallet_bloc.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class SignUpButton extends StatelessWidget {
+  const SignUpButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final width = MediaQuery.of(context).size.width;
+    final height = MediaQuery.of(context).size.height;
+    const textSize = 20.0;
+    return TextButton(
+      style: ButtonStyle(
+        backgroundColor: MaterialStateProperty.all<Color>(
+          Colors.amber[200]!.withOpacity(0.15),
+        ),
+        shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+          RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(20),
+            side: const BorderSide(color: Colors.white),
+          ),
+        ),
+        minimumSize: MaterialStateProperty.all<Size>(
+          Size(width / 4, height * 0.09),
+        ),
+        maximumSize: MaterialStateProperty.all<Size>(
+          Size(width / 2, height * 0.10),
```

Comment:
The size of the button should depend on the REMAINING size that is allocated by the widget tree rather than the entire size of the screen. Also avoid using [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) Use   [LayoutBuilder](https://api.flutter.dev/flutter/widgets/LayoutBuilder-class.html) instead.